lib/init.js

```diff
@@ -164,6 +164,8 @@ function prompt(config, detected, callback) {
     },
   ]).then((answers) => {
     callback(null, answers);
+  }).catch((error) => {
```

Comment:
I've noticed Promise rejection is not handled, thus the following line will never resolve in case of error:

https://github.com/apiaryio/dredd/blob/07c8c8157de47658d5e8f8f4c58ee10ea1f8ef21/lib/init.js#L25